repobee_sanitizer/_sanitize_repo.py

```diff
@@ -92,10 +92,38 @@ def sanitize_files(
     return []
 
 
+def create_pr_branch(
+    repo_path: pathlib.Path, target_branch: str, pr_branch_name: str
+):
+    """Create a new branch from the target_branch that we sanitize to
+    and a pull request can be created from.
+
+    Args:
+        repo_path: Path to the repository
+        target_branch: The path to create the pull request branch from
+        pr_branch_name: The name of the branch to sanitize to and create a pr
+        from
+    """
+
+    repo = git.Repo(str(repo_path))
+    source_branch = repo.head.ref.path
+
+    # Assures the target branch is not empty
+    try:
+        repo.git.symbolic_ref("HEAD", f"refs/heads/{target_branch}")
+        repo.index.diff("HEAD")
+    except git.BadName:
+        repo.git.commit("--allow-empty", "-m", "initial commit")
+
+    # Create the pr branch from the target branch
+    repo.git.branch(pr_branch_name, target_branch)
+
+    # Go back to the source branch
+    repo.git.checkout(source_branch)
```

Comment:
Avoid inline comments explaining what the code does. In general, if you need inline comments to explain _what_ the code does, the code should be refactored so as not to need them. Comments explaining _why_ the code does something are more ok, but should still be used with care.

But here, the code explains itself. If the reader of the code doesn't know what `git branch` and `git checkout` does, then they simply need to read up a bit on the domain.

```suggestion
    repo.git.branch(pr_branch_name, target_branch)
    repo.git.checkout(source_branch)
```